examples/lightning_base.py

```diff
@@ -240,7 +244,7 @@ def add_generic_args(parser, root_dir) -> None:
         help="For fp16: Apex AMP optimization level selected in ['O0', 'O1', 'O2', and 'O3']."
         "See details at https://nvidia.github.io/apex/amp.html",
     )
-    parser.add_argument("--n_tpu_cores", dest="tpu_cores", type=int, default=0)
+    parser.add_argument("--n_tpu_cores", dest="tpu_cores", type=int)
```

Comment:
why?

Reply:
See: error 2 in https://github.com/huggingface/transformers/pull/6028

Reply:
Yes, this started happening because of a PL thing, let me find that pull request from their repository

Reply:
Yeah hear it is:

`    if not _tpu_cores_valid(tpu_cores):
        raise MisconfigurationException("`tpu_cores` can only be 1, 8 or [<1-8>]")`

PyTorchLightning/pytorch-lightning#2094